src/bookmark.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bookmarked</title>
+  </head>
+  <body>
+    <header class="app-header">
+      <h1 class="app-header__heading">Magic Quizard</h1>
```

Comment:
I'd name it `Bookmark` or something like that so the user knows on which page he/she is.
```suggestion
      <h1 class="app-header__heading">Bookmark</h1>
```